src/pages/TradeV3/mobile/OrderAccountPositions.tsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Tabs, TabsContent, TabsList, TabsTrigger, cn } from 'gfx-component-lib'
+import React, { Dispatch, FC, ReactElement, SetStateAction } from 'react'
+import { TitleLabel } from '../perps/components/PerpsGenericComp'
+import { OrderBook } from '../OrderBook'
+import { RecentTrades } from '../RecentTrades'
+import PlaceOrderMobiV2 from './PlaceOrderMobiV2'
+import { CollateralPanel } from '../perps/components/CollateralPanel'
+import OpenOrdersAndPositions from './OpenOrdersAndPositions'
+
+const OrderAccountPositions: FC<{
+  tabs: string[]
+  selectedTab: string
+  setSelectedTab: Dispatch<SetStateAction<string>>
+}> = ({ tabs, selectedTab, setSelectedTab }): ReactElement => (
+  <div>
+    <div>
+      <Tabs className="p-[0px] sm:max-h-[450px] mb-24 sm:mb-4" defaultValue="1">
+        <TabsList className="sm:dark:!bg-black-1 sm:bg-grey-5 sm:py-[15px]">
+          <TabsTrigger
+            className={cn('w-[33%]')}
```

Comment:
could we potentially benefit from dynamic sizing to reduce the className's we have to type comp-lib side?